pytest_reportportal/plugin.py

```diff
@@ -27,13 +29,20 @@ def is_master(config):
 
 @pytest.mark.optionalhook
 def pytest_configure_node(node):
+    if node.config._reportportal_enabled is False:
```

Comment:
wasn't this supposed to be `_reportportal_configured` ? i can't see any reference to this attribute anywhere ...and my tests blow up on this :)

Reply:
Need to verify, I lost track a last a few months.